### PR TITLE
Add Carp::Clan perl module

### PR DESCRIPTION
**Pull request by fritzkink**

### CODE REVIEW

File: components/perl/Carp-Clan/Carp-Clan-PERLVER.p5m [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2022 Friedrich Kink
+#
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.human-version value=$(HUMAN_VERSION)
```
Comment (AndWac) [inline components/perl/Carp-Clan/Carp-Clan-PERLVER.p5m:17]: -$(PLV) is missing from the fmri.

File: components/perl/Carp-Clan/pkg5 [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+        "SUNWcs",
+        "runtime/perl-522",
+        "runtime/perl-524",
+        "runtime/perl-534",
+        "shell/ksh93",
+        "system/library"
+    ],
+    "fmris": [
+        "library/perl-5/carp-clan",
+    ],
```
Comment (AndWac) [inline components/perl/Carp-Clan/pkg5:15]: This is the result of the missing -$(PLV) on the fmri.

File: components/perl/Carp-Clan/Carp-Clan-PERLVER.p5m (hunk 1 above)
Comment (fritzkink) [reply components/perl/Carp-Clan/Carp-Clan-PERLVER.p5m:17]: Again thanks for pointing out. Fixed now.

File: components/perl/Carp-Clan/pkg5 (hunk 2 above)
Comment (fritzkink) [reply components/perl/Carp-Clan/pkg5:15]: also fixed by inheritance.

**Comment by AndWac** [comment]
You should also add (and probably normalize) test results.

**Review by AndWac — APPROVED** [review]

**Pull request merged**